tst/regression/CMakeLists.txt

```diff
@@ -76,6 +76,15 @@ if (ENABLE_HDF5)
   --num_steps 25")
   list(APPEND EXTRA_TEST_LABELS "")
 
+  # Particles example
+  list(APPEND TEST_DIRS particles)
+  list(APPEND TEST_PROCS 1) # Temporarily force scalar testing
```

Comment:
How temporary is this, otherwise we will essentially be running two tests that are doing the same thing? Also how long does this test actually take to run?

Reply:
My next task with this particles implementation is to add MPI communication, so this should be temporary only until the next particles PR (though that may take a little while to put together). This test runs in less than a second on my laptop, so I don't think this will cause any issues? Though I might increase the particle number a bit for good measure. I didn't see another simple way to enroll a regression test as scalar-only.

Reply:
There are `mpi-yes` and `mpi-no` so for a "scalar" test the `mpi-no` label should be sufficient that the test is only run when not compiled for MPI.